prody/sequence/analysis.py

```diff
@@ -992,9 +993,15 @@ def alignSequenceToMSA(seq, msa, **kwargs):
             ag = seq.select('chain {0}'.format(chain))
```

Comment:
We should probably have a try except here. Someone could provide a string that isn't valid for selecting chains.